### PR TITLE
Add usage of i18n for plugins manager

### PR DESCRIPTION
**Pull request by bbogdanov**
Forgot to use i18n for the labels

### CODE REVIEW

File: src/components/dialogs/searchDialog/PluginsManager.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
     scrollable
   >
     <v-card>
-      <v-card-title> <v-icon>mdi-toy-brick</v-icon> Plugins manager </v-card-title>
+      <v-card-title> <v-icon>mdi-toy-brick</v-icon> {{ $t("plugin", 1) }} {{ $t("manager") }} </v-card-title>
```
Comment (CzBiX) [inline src/components/dialogs/searchDialog/PluginsManager.vue:8]: can be one label

File: src/components/dialogs/searchDialog/SearchDialog.vue [hunk 2]
```diff
@@ -50,7 +50,7 @@
         </v-card-text>
         <v-card-actions>
           <v-btn @click="openPluginManager">
-            <v-icon>mdi-cog</v-icon>  Plugins manager
+            <v-icon>mdi-cog</v-icon> {{ $t("plugin", 1) }} {{ $t("manager") }}
```
Comment (CzBiX) [inline src/components/dialogs/searchDialog/SearchDialog.vue:53]: also

File: src/locale/en.ts [hunk 3]
```diff
@@ -64,6 +64,7 @@ export default {
   action: 'Action |||| Actions',
   search_engine: 'Search engine',
   usage: 'usage',
+  manager: 'manager',
```
Comment (CzBiX) [inline src/locale/en.ts:67]: also

**Review by CzBiX — CHANGES_REQUESTED** [review]

**Comment by bbogdanov** [comment]
@CzBiX Do not forget to squash

**Pull request merged**